Modules/System/Base64 Convert/src/Base64Convert.Codeunit.al

```diff
@@ -71,6 +107,35 @@ codeunit 4110 "Base64 Convert"
         exit(Base64ConvertImpl.FromBase64(Base64String));
     end;
 
+    /// <summary>
+    /// Converts the specified string, which encodes binary data as base-64 digits, to an equivalent regular string.
+    /// </summary>
+    /// <param name="Base64String">The string to convert.</param>
+    /// <param name="TextEncoding">The TextEncoding for the input string.</param>
+    /// <returns>Regular string that is equivalent to the input base-64 string.</returns>
+    /// <error>The length of Base64String, ignoring white-space characters, is not zero or a multiple of 4.</error>
+    /// <error>The format of Base64String is invalid. Base64String contains a non-base-64 character, more than two padding characters,
```

Comment:
The format of Base64String is not valid. This can be because Base64String contains a non-base-64 character, more than two padding characters, or a non-white space character in the padding characters.

Reply:
Again, just a copy'paste of your existing documentation.